dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -429,51 +518,53 @@ private void ds3TransferToLocal() {
                 } else {
                     localPath = selectedAtDest.getPath();
                 }
-
-                final String priority = (!savedJobPrioritiesStore.getJobSettings().getGetJobPriority().equals(resourceBundle.getString("defaultPolicyText"))) ? savedJobPrioritiesStore.getJobSettings().getGetJobPriority() : null;
-
+                final String priority = (!savedJobPrioritiesStore.getJobSettings().getGetJobPriority()
+                        .equals(resourceBundle.getString("defaultPolicyText"))) ?
+                        savedJobPrioritiesStore.getJobSettings().getGetJobPriority() : null;
                 final Ds3GetJob getJob = new Ds3GetJob(selectedItemsAtSourceLocationListCustom, localPath, session.getClient(),
-                        deepStorageBrowserPresenter, priority, settingsStore.getProcessSettings().getMaximumNumberOfParallelThreads(), jobInterruptionStore, ds3Common);
+                        deepStorageBrowserPresenter, priority, settingsStore.getProcessSettings().getMaximumNumberOfParallelThreads(),
+                        jobInterruptionStore, ds3Common);
                 jobWorkers.execute(getJob);
-
                 getJob.setOnSucceeded(event -> {
                     LOG.info("Succeed");
-                    refreshLocalSideView(selectedItemsAtDestination, treeTable, fileRootItemLabel, fileRootItem);
+                    refreshLocalSideView(selectedItemsAtDestination, treeTable, localFilePathIndicator, fileRootItem);
                 });
-
                 getJob.setOnFailed(e -> {
-                    LOG.error("Get Job failed");
-                    refreshLocalSideView(selectedItemsAtDestination, treeTable, fileRootItemLabel, fileRootItem);
+                    LOG.info("Get Job failed");
+                    refreshLocalSideView(selectedItemsAtDestination, treeTable, localFilePathIndicator, fileRootItem);
                 });
-
                 getJob.setOnCancelled(e -> {
                     LOG.info("Get Job cancelled");
                     if (getJob.getJobId() != null) {
                         try {
                             session.getClient().cancelJobSpectraS3(new CancelJobSpectraS3Request(getJob.getJobId()));
                             ParseJobInterruptionMap.removeJobID(jobInterruptionStore, getJob.getJobId().toString(), getJob.getDs3Client().getConnectionDetails().getEndpoint(), deepStorageBrowserPresenter);
-                            deepStorageBrowserPresenter.logText("GET Job Cancelled", LogType.ERROR);
-                        } catch (final IOException e1) {
+                            deepStorageBrowserPresenter.logText(resourceBundle.getString("getJobCancelled"), LogType
+                                    .ERROR);
+                        } catch (final IOException ie1) {
```

Comment:
Unless there is extra logic being added by adding the new Exception catch block, the IOException catch block should be removed.